community/kernel/src/main/java/org/neo4j/kernel/AvailabilityGuard.java

```diff
@@ -281,6 +281,7 @@ private Availability availability( long millis )
             return availability;
         }
 
+        log.warn( "Database is unavailable awaiting for it to be available again for " + millis + "ms" );
```

Comment:
It just didn't feel like something that needed to be a cause for alarm, it's pretty "normal" for wait to happen, is it not?